pkg/github/projects.go

```diff
@@ -877,21 +916,53 @@ type projectV2Field struct {
 	UpdatedAt *github.Timestamp `json:"updated_at,omitempty"` // The time when this field was last updated.
 }
 
+type projectV2ItemFieldValue struct {
+	ID       *int64      `json:"id,omitempty"`        // The unique identifier for this field.
+	Name     string      `json:"name,omitempty"`      // The display name of the field.
+	DataType string      `json:"data_type,omitempty"` // The data type of the field (e.g., "text", "number", "date", "single_select", "multi_select").
+	Value    interface{} `json:"value,omitempty"`     // The value of the field for a specific project item.
+}
+
 type projectV2Item struct {
-	ID            *int64            `json:"id,omitempty"`
-	Title         *string           `json:"title,omitempty"`
-	Description   *string           `json:"description,omitempty"`
-	NodeID        *string           `json:"node_id,omitempty"`
-	ProjectNodeID *string           `json:"project_node_id,omitempty"`
-	ContentNodeID *string           `json:"content_node_id,omitempty"`
-	ProjectURL    *string           `json:"project_url,omitempty"`
-	ContentType   *string           `json:"content_type,omitempty"`
-	Creator       *github.User      `json:"creator,omitempty"`
-	CreatedAt     *github.Timestamp `json:"created_at,omitempty"`
-	UpdatedAt     *github.Timestamp `json:"updated_at,omitempty"`
-	ArchivedAt    *github.Timestamp `json:"archived_at,omitempty"`
-	ItemURL       *string           `json:"item_url,omitempty"`
-	Fields        []*projectV2Field `json:"fields,omitempty"`
+	ID            *int64                     `json:"id,omitempty"`
+	Title         *string                    `json:"title,omitempty"`
+	Description   *string                    `json:"description,omitempty"`
+	NodeID        *string                    `json:"node_id,omitempty"`
+	ProjectNodeID *string                    `json:"project_node_id,omitempty"`
+	ContentNodeID *string                    `json:"content_node_id,omitempty"`
+	ProjectURL    *string                    `json:"project_url,omitempty"`
+	ContentType   *string                    `json:"content_type,omitempty"`
+	Creator       *github.User               `json:"creator,omitempty"`
+	CreatedAt     *github.Timestamp          `json:"created_at,omitempty"`
+	UpdatedAt     *github.Timestamp          `json:"updated_at,omitempty"`
+	ArchivedAt    *github.Timestamp          `json:"archived_at,omitempty"`
+	ItemURL       *string                    `json:"item_url,omitempty"`
+	Fields        []*projectV2ItemFieldValue `json:"fields,omitempty"`
+}
```

Comment:
similar to this [comment](https://github.com/github/github-mcp-server/pull/1282/files#r2453585305) for the remaining structs, we should also capitalize the first letter as previously mentioned imo as to limit the diff from `google/go-github`

Reply:
Replied above.